doc/sphinx/source/get-started/installation.rst

```diff
@@ -174,12 +174,12 @@ explained above, if the user has not already done so.
    .. code::
 
        conda install pkg-config swig cmake
-      
-   When working on a Linux system it is `currently 
+
+   When working on a Linux system it is `currently
    <https://github.com/NNPDF/nnpdf/pull/1280>`_ also needed to run
-   
+
    .. code::
-   
+
```

Comment:
This is in the diff just by change, but this whole thing can be removed if you feel like it (many other things are outdated so... just because it appeared here)

Reply:
Yes at least these instructions should be up to date. I'll do this in a different PR